draft-ietf-dnsop-svcb-https.md

```diff
@@ -1212,12 +1212,14 @@ Consider a simple zone of the form:
 
 The domain owner could add this record:
 
-    simple.example. 7200 IN HTTPS 1 . alpn=h3
+    @ 7200 IN HTTPS 1 . alpn=h3
 
-to indicate that simple.example uses HTTPS, and supports QUIC
+to indicate that https://simple.example uses HTTPS, and supports QUIC
```

Comment:
```suggestion
to indicate that https://simple.example (and http://simple.example)
use HTTPS, and supports QUIC
```

Reply:
I added a sentence about this at the end of the example.